### DISCUSSION TITLE
Bug or bad use? when I edit my posts, sometimes, the images galeries have disappeared 😯!

### DISCUSSION THREAD

**Discussion by Vinc26** [GetPublii/Publii#1862] [Help]
Hi there,

Three days after published my website, I decide tonight to correct some spelling mistakes, add a photo...
And after publication: all the photos on the modified page have disappeared! Those presented in the form of galleries! Luckily I kept the originals in a corner. But what a job to have to look for everything, and put it back in order.

Is it a bug? A known bug? Or am I using Publii wrong? Something I misunderstood?

To clarify: I use the "blocks" mode, and these are the images presented in the form of galleries. On the editing page, the images seem to be there: I see them. On the other hand, when I choose to edit the galerie block, for example to add an image, the thumbnails are finally absent, replaced by a broken mini image. Finally, note that I use the automatic WEBP conversion proposed by Publii ; But I don't know if that's the cause of that.

Help! I don't dare to edit my pages anymore! 😬

**Comment by michaelfluegge** [comment]
It is hard to tell by your description, but I assume, it is a kind of doing something wrong while adding or changing things. Maybe try using the WYSIWYG for a while and see, whether the images break or not.

**Reply by Vinc26** [reply]
I think it's not a bad manipulation when editing the article. Because all posts and pages are affected! As soon as I edit a post or a page, all the images in his galleries are absent, and upon publication, all have disappeared. It's the same for all articles on the site.

It looks like Publii can't find the originals. And yet, I didn't move anything or rename anything at all.

**Reply by shawngiese** [reply]
Maybe check the browser developer console to check:

- the URL to the images
- if the images are being requested as WEBP or in their original format
- if there are any console errors

Also, what kinds of names do the images have? I noticed that the file manager had problems loading images with long names. 

Worst case, export everything to a zip file and then go into it and check that everything is deployed as expected.

Just some ideas to maybe help pinpoint where the problem is.

**Reply by Vinc26** [reply]
I tried to understand something in the Publii Tools > Log Viewer... 😬 The only error log with content is `rendering error log` :

```
(node:42092) Warning: No such label 'BLOG PAGINATION' for console.timeEnd()
(Use `Publii Helper --trace-warnings ...` to show where the warning was created)

```

About the browser developer console: 
When I write those lines, I can't reproduce the bug! So... next time 🤷‍♂️

**Comment by candidexmedia** [comment]
Were you able to solve this? What theme are you using?

**Reply by Vinc26** [reply]
Hi, no it's not solved. I didn't do anything to correct it, but currently, it doesn't do it anymore (I've tried to edit a post, and it almost work: there is small breaking images instead of thumbnail, but when publish, all is ok. See joined screenshot).
But I feel like it can do it at any time. It's already been like that. I had seen this problem while developing the website, then no more problem, then again the problem, and now it is (almost) no longer there. 🤷‍♂️🥴

<img width="890" alt="Capture d’écran 2024-12-13 à 09 06 32" src="https://github.com/user-attachments/assets/339069f4-71ac-4990-a1b1-b8dbec00f088" />

**Reply by candidexmedia** [reply]
That's very odd... Which version of the Publii app are you using, and on which operating system?

**Reply by Vinc26** [reply]
Publii 0.46.2 , macOS 15.1.1 , on a MacBook Air M2 , using the theme Simple.
